src/autoscaler/configutil/cf.go

```diff
@@ -0,0 +1,194 @@
+package configutil
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+
+	"code.cloudfoundry.org/app-autoscaler/src/autoscaler/models"
+	"github.com/cloudfoundry-community/go-cfenv"
```

Comment:
This looks really ancient und unmaintained. Does the fork at https://github.com/cloud-gov/go-cfenv work?

Reply:
fixed in 348425906